etc/methods.yaml

```diff
@@ -211,7 +211,18 @@ eth_getTransactionByBlockNumberAndIndex:
   implementations:
     - etc/schema/functions/eth_getTransactionByBlockNumberAndIndex.sql
     - src/servers/skeleton.ts
-
+eth_getTransactionByBlockNumber:
```

Comment:
`eth_getTransactionByBlockNumber` should be `eth_getTransactionsByBlockNumber`

Reply:
for visibility - Ahmed means Transaction`s` as plural